communicator_node:
  ros_parameters:
    robots:
      Husky1:
        type: Husky
        topics:
          odometry: /transformed_odom
          navigation:
            navigation_path: e.g. /Husky1/transformed_global_plan
            navigation_status: e.g. /Husky1/navigate_to_pose/_action/status
          feedback:
            robot_status: /husky/status

      Kobuki1:
        type: Kobuki
        topics:
          odometry: /slam_odom
          navigation:
            navigation_path: /transformed_global_plan
            navigation_status: /navigate_to_pose/_action/status
          feedback:
            battery_state: /battery_state
            wheel_vel: /wheel_vels
            navigation_path: /plan 
      
      Locobot1:
        type: Locobot
        topics:
          odometry: e.g. /odom
          navigation:
            navigation_path: e.g. /transformed_global_plan
            navigation_status: e.g. /navigate_to_pose/_action/status
          feedback:
            joint_states: /wx200/joint_states
            battery_state: /locobot1/battery_state
            wheel_vel: /locobot1/wheel_vels